tests/test_save.py

```diff
@@ -100,21 +117,22 @@ def test_model_manipulation(request, model_class, storage_method, store_format):
         model.set_env(env)
 
         # predict the same output before saving
-        env.envs[0].action_space.seed(0)
-        loaded_mean_reward, _ = evaluate_policy(model, env, deterministic=True, n_eval_episodes=N_EVAL_EPISODES)
+        # TODO: use pre-defined observations instead of reward
```

Comment:
Removed because unstable test, I assume? I would like to see something replace it (will probably never be fixed otherwise ^^), but I have no good idea for this one

Reply:
yes, a proper test would be to do what is done in SB3: https://github.com/DLR-RM/stable-baselines3/blob/master/tests/test_save_load.py#L154